server/src/services/exercisePresetService.ts

```
@@ -0,0 +1,74 @@
+import { error } from "console";
+import { exercisePresets } from "../models/exercisePresetModel";
+
+
+export class ExercisePresetService {
+    async addExercise(data: any) {
+        try {
+
+            const exerciseExists = await exercisePresets.count(data)
+            if (Boolean(exerciseExists)) {
+                throw new Error(`אי אפשר להוסיף פריט שכבר קיים לרשימה!`);
+            }
+
+
+            const exercise = await exercisePresets.create(data)
+
+            return exercise
+
+        } catch (error) {
+            return Promise.reject(error)
+        }
+    }
+
+    async getExercises() {
+        try {
+
+            const exercises = await exercisePresets.find()
+
+            return exercises
+
+        } catch (error) {
+            return error
```

Change all of these to throws instead of returns. Also in general no spaces when starting try blocks or after return statements. (Only if it is the final return statement. If there is code after it is fine to put a space